common/src/main/java/com/microsoft/identity/common/internal/util/StringUtil.java

```diff
@@ -135,6 +137,15 @@ public static String join(final char delimiter, @NonNull final Iterable<String>
                 && !StringExtensions.isNullOrBlank(uidUtidArray[INDEX_UTID])) {
             uid = uidUtidArray[INDEX_UID];
             utid = uidUtidArray[INDEX_UTID];
+        } else {
```

Comment:
This method has been migrated to 4j (same class name), and this change should go there as well.

Reply:
added